core/templates/pages/login-page/login-page.import.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 The Oppia Authors. All Rights Reserved.
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Why was this changed? This file was created in 2021

Reply:
reverted